### PR TITLE
Enhancements to JavaScript and Python 

### PR DESCRIPTION
**Pull request by jmasonlee**
This pull request was done with @isidore to improve the setup for python and node based on what we learned in the learning hour. It includes:

- run_test.sh scripts 
- The necessary python module __init__.py file in src
- An example of importing Calculator in node
- Enhanced documentation for PyCharm

### CODE REVIEW

File: javascript/run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
```
Comment (JayBazuzi) [inline javascript/run_tests.sh:2]:
```suggestion
set -e

```

File: python/run_tests.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
```
Comment (JayBazuzi) [inline python/run_tests.sh:2]:
```suggestion
set -e

```

File: javascript/tests/calculotor.test.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 const test = require('tape')
+const Calculator = require('../calculator')
 
 test('should not throw when authorized', (t) => {
-  // TODO: write a test that fails due to the bug in
-  // Calculator.divide()
+  const calculator = new Calculator()
```
Comment (emilybache) [inline javascript/tests/calculotor.test.js:5]: This is too much code for the starting position

**Comment by emilybache** [comment]
Thanks for making it easier for people using these languages to run the tests. Just a couple of small things to fix, hope it's ok?

**Comment by codecop** [comment]
@jmasonlee are you able to make the required changes and resolve merge conflicts or do you need help?

File: python/src/__init__.py
Comment (emilybache) [inline python/src/__init__.py:1]: I believe this file shouldn't be needed, 'src' is not a module it's a source folder you put on your PYTHONPATH.

**Comment by codecop** [comment]
@emilybache I prepared a new PR with required fixes in #4 , this PR can be closed.

**Pull request closed without merging**